tests/test_api.py

```diff
@@ -1,88 +0,0 @@
-"""Tests for Storj api."""
-
-import asyncio
-
-import aiohttp
-from custom_components.storj.api import (
-    StorjClient,
-)
-from homeassistant.helpers.aiohttp_client import async_get_clientsession
-
-
-async def test_api(hass, aioclient_mock, caplog):
```

Comment:
Another reason for removing this is that the other backup integrations that I'm modeling this one after don't test the API directly - they have a `test_backup` file.